SwiftWisdom/Core/UIKit/NibLoadableView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NibLoadableView.swift
+//  SwiftWisdom
+//
+//  Created by Christian Howland on 3/31/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+/**
+ Subclassing NibLoadableView allows you to automate the loading of a Nib into a placeholder view inside
+ another Nib, using only the Custom Class attribute.
+ 
+ It removes the need to perform the following manual steps every time you want a Nib that can load into a placeholder View:
+
+ - Setting an outlet for the top level view
+ - Setting 'translatesAutoresizingMaskIntoConstraints' to false
+ - Adding constraints to fit the new view into its container
+ - Writing boilerplate Nib loading code.
+
+ The minimum amount of code required to use this would just be the inheritance line, i.e.:
+
+ " YourCustomView: NibLoadableView {} "
```

Comment:
@brightredchilli that sounds like a really good solution -- my only concern with that approach is that you wouldn't be able to use IBDesignables, since the Custom Class you set isn't the 'actual' view that will ultimately be loaded in. In my eyes, an ideal solution has 2 requirements: 1) keep IBDesignable functionality intact and 2) eliminating the need to do all the manual nib loading and adding of constraints. 

Thoughts?